README.md

```diff
@@ -62,7 +62,8 @@ and health-checking mechanisms.
 * Install py-utils.
 
    Please refer to [the instruction](https://github.com/Seagate/cortx-utils/blob/main/py-utils/README.md) to install corxt-py-utils from sources.
-
+   
+   Note: `Hare` needs `pip` and `RPM` packages, so please [build](https://github.com/Seagate/cortx-utils/blob/main/py-utils/README.md#build) both packages. Then [install](https://github.com/Seagate/cortx-utils/blob/main/py-utils/README.md#installation) both of them using `pip3 install` and `yum install` respectively. 
```

Comment:
@knekrasov ,
Do we need any changes in requirements.txt, hare.spec files to avoid these additional steps?

Reply:
Honestly speaking, I don't quite get the point about `pip`.

Hare does require pip. That requirement is already expressed via `hare.spec`:
```
BuildRequires: python36-pip
```
In other words, the package is required to build Hare. When the user just installs from the RPM it, pip is not required.

Suggestion:
Change line 52 of README.md:
```
  sudo yum -y install python3 python3-devel
```
to the following:
```
  sudo yum -y install python36 python36-devel python36-pip
```

Reply:
Hi @knekrasov , thanks for the suggestions, I will test this out. Also, I can show you how to reproduce this problem easily. Please let me know if you would be interested in an online demo. Frankly, I have little knowledge of how `pip` causes the bug to appear, but I am happy to help as much as I could. Thanks

Reply:
@knekrasov , I tried your suggestion, but there is no effect and the problems still persist. I can give you access to my cluster to try out the Readme. Let me know if I should provide more proof to justify this PR. 
cc: @suvratjoshi @vaibhavparatwar

Reply:
Hi @knekrasov, I just realized that you might misunderstand the problem. What do I mean by `2. If we only install pip package`, is not about installing `pip`, but it is referring to the [cortx-utils Readme](https://github.com/Seagate/cortx-utils/blob/main/py-utils/README.md#build) where it shows you how to build + install the package using `pip`, specifically, this is the script to do so:
```sh 
$ pip3 install wheel
$ cd ./cortx-utils/py-utils
$ python3 setup.py bdist_wheel
```

The problem that I address on this PR is not about the pip requirement, but the cortx-py-utils installed by `pip`. Please see the `Files changed` tab to understand the modification. I'd love to close (reject or merge) this PR soon so that I can focus on another test. I appreciate if you could check this PR again. Thx! :)

Reply:
I see. Thank you for all your efforts!